### DISCUSSION TITLE
Lively Wallpaper continues to play the animated background with windows in focus over another app in full screen mode

### DISCUSSION THREAD

**Discussion by vindictaBit** [rocksdanister/lively#1687] [Q&A] [answered]
I have been seeing that the app plays the background when using a windowed app over another app that is already in full screen. Instead it works fine and plays nothing only when I focus a full screen app.

I already tried activating the full screen apps option and the screen pause rule to all screens but it's still the same.

So, what can I do so that lively wallpaper doesn't play the background when I have windows over an application in full screen?

**Comment by rocksdanister** [comment, marked as answer]
This is the case since there can only be one foreground application at a time (Foreground algorithm.)

Setting the following will pause the wallpaper if any program is in focus (not on desktop.):
`Wallpaper playback -> Application Focused -> Pause`

Related https://github.com/rocksdanister/lively/issues/474

**Comment by studeimus** [comment]
Hi. I have this same problem, and have pegged back most of the possibly related LW options (including the one above), to no avail. Am using the Fluids wallpaper (disabling all funky gfx makes no difference).

I have 3x monitors, two of which are through a single USB-C cable, and with each monitor with a full-screen app running, the GPU sits pegged at about 65%. CPU is practically idle.

System is a Surface Laptop 4, with Intel Irix Xe Graphics, and the two external screens are at 1080P resolution.

Thoughts? I realise this is not a very gfx capable system, but then in this scenario, I don't expect it to have ANY gfx load.

Stu.

**Reply by rocksdanister** [reply]
https://github.com/rocksdanister/lively/wiki/Performance
